seed/challenges/08-coding-interview-prep/rosetta-code.json

```diff
@@ -3954,6 +3954,175 @@
+      "solutions": ["function gray(enc,number){\n  if(enc){\n      return number ^ (number >> 1);\n  }else{\n      let n = number;\n\n      while (number >>= 1) {\n          n ^= number;\n      }\n      return n;\n  }\n}\n"],
+      "tests": [{
+        "text": "'<code>gray</code> should be a function.'",
+        "testString": "assert(typeof gray=='function','<code>gray</code> should be a function.');"
+      }, {
+        "text": "'<code>gray(true,177)</code> should return a number.'",
+        "testString": "assert(typeof gray(true,177)=='number','<code>gray(true,177)</code> should return a number.');"
+      }, {
+        "text": "'<code>gray(true,177)</code> should return <code>233</code>.'",
+        "testString": "assert.equal(gray(true,177),233,'<code>gray(true,177)</code> should return <code>233</code>.');"
+      }, {
+        "text": "'<code>gray(true,425)</code> should return <code>381</code>.'",
+        "testString": "assert.equal(gray(true,425),381,'<code>gray(true,425)</code> should return <code>381</code>.');"
+      }, {
+        "text": "'<code>gray(true,870)</code> should return <code>725</code>.'",
+        "testString": "assert.equal(gray(true,870),725,'<code>gray(true,870)</code> should return <code>725</code>.');"
+      }, {
+        "text": "'<code>gray(false,233)</code> should return <code>177</code>.'",
+        "testString": "assert.equal(gray(false,233),177,'<code>gray(false,233)</code> should return <code>177</code>.');"
+      }, {
+        "text": "'<code>gray(false,381)</code> should return <code>425</code>.'",
+        "testString": "assert.equal(gray(false,381),425,'<code>gray(false,381)</code> should return <code>425</code>.');"
+      }, {
+        "text": "'<code>gray(false,725)</code> should return <code>870</code>.'",
+        "testString": "assert.equal(gray(false,725),870,'<code>gray(false,725)</code> should return <code>870</code>.');"
+      }],
+      "id": "5a23c84252665b21eecc7e80",
+      "challengeType": 5,
+      "releasedOn": "June 7, 2018",
+      "files": {
+        "indexjs": {
+          "key": "indexjs",
+          "ext": "js",
+          "name": "index",
+          "contents": ["function gray (enc,number) {", "  // Good luck!", "}"],
```

Comment:
Could you change this line to the following?:

"contents": ["function gray (enc, number) {", "  // Good luck!", "}"],